soql-server-pg/src/main/scala/com/socrata/pg/server/QueryServer.scala

```diff
@@ -431,8 +431,19 @@ class QueryServer(val dsInfo: DSInfo, val caseSensitivity: CaseSensitivity) exte
     }
   }
 
-  private def copyInfoHeader(copyNumber: Long, dataVersion: Long, lastModified: DateTime) = {
-    Header("Last-Modified", lastModified.toHttpDate) ~>
+
+  private def copyInfoHeaderForSchema(copyNumber: Long, dataVersion: Long, lastModified: DateTime) = {
+    copyInfoHeader("Last-Modified")(copyNumber, dataVersion, lastModified)
+  }
+
+  private def copyInfoHeaderForRows(copyNumber: Long, dataVersion: Long, lastModified: DateTime) = {
+    // TODO: Keeping "Last-Modified" just to make roll out easy.  It will be removed in the next cycle.
+    copyInfoHeader("Last-Modified")(copyNumber, dataVersion, lastModified)
+    copyInfoHeader("X-SODA2-Secondary-Last-Modified")(copyNumber, dataVersion, lastModified)
```

Comment:
This creates a response which will set last-modified, and then throw it away.  You need to combine these with `~>` to get both.